src/symbols.c

```diff
@@ -1704,13 +1704,17 @@ static gboolean goto_tag(const gchar *name, gboolean definition)
 }
 
 
-gboolean symbols_goto_tag(const gchar *name, gint pos, gboolean definition)
+gboolean symbols_goto_tag(GeanyDocument *doc, gint pos, gboolean definition)
 {
-	GeanyDocument *doc = document_get_current();
+	const gchar *name;
 
 	if (plugin_extension_goto_provided(doc, NULL))
 		return plugin_extension_goto_perform(doc, pos, definition);
 
+	editor_find_current_word(doc->editor, pos,
+		editor_info.current_word, GEANY_MAX_WORD_LENGTH, NULL);
+	name = *editor_info.current_word ? editor_info.current_word : NULL;
+
```

Comment:
Any reason not to include the `if (! name) return FALSE;`?  Apparently `goto_tag()` seems NULL-safe, but we'll still end up with updating the statusbar with a message including a NULL representation.

Reply:
Yeah, midnight, my enemy :-).

Reply:
Hehe :)  I'll trun in myself, hopefully not letting you hanging too much, but I'll stop being of any use pretty soon :slightly_smiling_face: